frontend/components/navigation-bar/index.tsx

```diff
@@ -38,23 +38,28 @@ const NavigationBar: React.FC = () => {
     const isMobile = useIsMobile();
 
     return(
-        <StyledAppBar>
-            <Toolbar>
-                <StyledGrid mobile={isMobile.toString()}>
-                    <StyledLeftContent>
-                        <StyledLogo />
-                        {!isMobile && <SearchBar />}
-                    </StyledLeftContent>
-                    <StyledRightContent mobile={isMobile.toString()}>
-                        {isMobile ? 
-                                <SearchBar /> 
-                            : 
-                                <NavigationTabs variant={TabsVariant.Standard} />
-                        }
-                    </StyledRightContent>
-                </StyledGrid>
-            </Toolbar>
-        </StyledAppBar>
+        <>
+            <StyledAppBar>
+                <Toolbar>
+                    <StyledGrid mobile={isMobile.toString()}>
+                        <StyledLeftContent>
+                            <StyledLogo />
+                            {!isMobile && <SearchBar />}
+                        </StyledLeftContent>
+                        <StyledRightContent mobile={isMobile.toString()}>
+                            {isMobile ? 
+                                    <SearchBar /> 
+                                : 
+                                    <NavigationTabs variant={TabsVariant.Standard} />
+                            }
+                        </StyledRightContent>
+                    </StyledGrid>
+                </Toolbar>
+            </StyledAppBar>
+
+            {/* Ugly fix to prevent appbar hidding content beneath it */}
```

Comment:
? there must be a better way

Reply:
Tried to look for a better way, every other options were over complicated. See [this article](https://stackoverflow.com/questions/48508449/content-beneath-fixed-appbar). Changing the position to `sticky` didn't work for us. The other accepted solution is complicated for our use case (app bar is a component).

Reply:
Ok we can keep it like this for now, could you open an issue to investigate further when we will have time? 😊

Reply:
Sure!